configure_posix.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cmake -B build tests \
+-DCONFIG_MENDER_PLATFORM_FLASH_TYPE="posix" \
+-DCONFIG_MENDER_PLATFORM_LOG_TYPE="posix" \
+-DCONFIG_MENDER_PLATFORM_NET_TYPE="generic/curl" \
+-DCONFIG_MENDER_PLATFORM_SCHEDULER_TYPE="posix" \
+-DCONFIG_MENDER_PLATFORM_STORAGE_TYPE="posix" \
+-DCONFIG_MENDER_PLATFORM_TLS_TYPE="generic/mbedtls" \
+-DCONFIG_MENDER_CLIENT_ADD_ON_INVENTORY=ON \
+-DCONFIG_MENDER_CLIENT_ADD_ON_CONFIGURE=OFF \
+-DCONFIG_MENDER_CLIENT_CONFIGURE_STORAGE=OFF \
+-DCONFIG_MENDER_CLIENT_ADD_ON_TROUBLESHOOT=OFF \
+-DCONFIG_MENDER_FULL_PARSE_ARTIFACT=ON \
+-DCONFIG_MENDER_PROVIDES_DEPENDS=ON
```

Comment:
Can you please rebase and add this option? :) 
```suggestion
-DCONFIG_MENDER_PROVIDES_DEPENDS=ON \
-DCONFIG_MENDER_ALL_WARNINGS_AS_ERRORS=ON
```